### PR TITLE
Added vue-cursor-fx to Miscellaneous

### PR DESCRIPTION
**Pull request by LuXDAmore**
An animated custom cursor effects for interactive elements like navigation, galleries and carousels.

### CODE REVIEW

**Comment by LuXDAmore** [comment]
fix: git: no conflicts resolved
fix: lists: not the last on the list

File: README.md [hunk 1]
```diff
@@ -1623,14 +1623,16 @@ Tooltips / popovers
  - [vue-insomnia](https://github.com/gorbypark/vue-insomnia) - Prevent display screen from sleeping (wake lock).
  - [vue-car-plate-keyboard](https://github.com/yimocanxue/vue-car-plate-keyboard) - A car license plate number keyboard for VueJS 2.x.(车牌号输入键盘，支持新能源车牌🚗🚗🚗)
  - [vue-dataflow-editor](https://github.com/krthr/vue-dataflow-editor) - Vue2 dataflow graph editor
+ - [cool-emoji-picker](https://github.com/kevinfaguiar/cool-emoji-picker) - A fast plug-n-play [Tw]emoji Picker (+textarea for Twemoji rendering) component for Vue.
```
Comment (Scrum) [inline README.md]: @LuXDAmore Needs to be at the end of the list.

Comment (LuXDAmore) [reply README.md]: I've putted it at the end, but i think that every time there is a new merge, my line is moved up

Comment (Scrum) [reply README.md]: This is already on the list.

**Pull request merged**